browser/brave_wallet/brave_wallet_service_unittest.cc

```diff
@@ -352,7 +355,8 @@ class BraveWalletServiceUnitTest : public testing::Test {
   mojom::BlockchainTokenPtr GetEthToken() { return eth_token_.Clone(); }
   mojom::BlockchainTokenPtr GetBatToken() { return bat_token_.Clone(); }
 
-  PrefService* GetPrefs() { return profile_->GetPrefs(); }
+  PrefService* GetProfilePrefs() { return profile_->GetPrefs(); }
+  TestingPrefServiceSimple* GetLocalState() { return local_state_->Get(); }
```

Comment:
Can we keep name of GetPrefs()? Renaming doesn't add much but makes diff quite heavy to review.
And in other test files too.

Reply:
So I suggest doing all sorts of GetPrefs->GetProfilePrefs refactoring in separate commit or separate PR.

Reply:
hmm... unsure about keeping the name as it may cause confusion in the future, but i'll be happy to split the renaming into a separate commit

Reply:
Normally `GetPrefs` is already implied getting prefs from profile, there is no need to specify `GetProfilePrefs`.
Take a look at upstream `ChromeAutocompleteProviderClient`